azurerm/internal/services/apimanagement/api_management_subscription_resource.go

```diff
@@ -184,6 +192,7 @@ func resourceArmApiManagementSubscriptionRead(d *schema.ResourceData, meta inter
 	d.Set("subscription_id", subscriptionId)
 	d.Set("resource_group_name", resourceGroup)
 	d.Set("api_management_name", serviceName)
+	d.Set("allow_tracing", resp.AllowTracing)
```

Comment:
rather than using `resp` here, we need to access this from within the `SubscriptionContractProperties` object (`properties` in the response) which could be nil - so can we move this inside the if statement below and make this:

```suggestion
		d.Set("allow_tracing", props.AllowTracing)
```

Reply:
Okay, I've updated the reading to be in the if statement and use the props object.